kpops/component_handlers/helm_wrapper/helm.py

```diff
@@ -192,8 +190,10 @@ def template(
             ]
             command.extend(flags.to_command())
             output = self.__execute(command)
+            if output == "":
+                return ()
```

Comment:
what does `KubernetesManifest.from_yaml(output)` produce without handling this special case?

Reply:
also I am wondering in which case a Helm template would render nothing at all?!

Reply:
It was my bad. We don't need this special case at all :)